### PR TITLE
835: rename repository

### PR DESCRIPTION
**Pull request by jorgesanchezperez**
- Renamed UI applications
- Readme files updated
- Shared components published with @secureapigateway scope version 1.0.0
- RCS updated with new secureapigateway components
- Make file updated

Issue: https://github.com/secureapigateway/secureapigateway/issues/835

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 blank_issues_enabled: false
 contact_links:
   - name: Developers and Collaborators guidelines
-    url: https://github.com/SecureBankingAccessToolkit/SecureBankingAccessToolkit/wiki/Developers-guideline
+    url: https://github.com/https://github.com/SecureApiGateway/SecureApiGateway/wiki/Developers-guideline
     about: Please read the developers and collaborators guideline before submit an issue.
   - name: Creation Issue page
-    url: https://github.com/SecureBankingAccessToolkit/SecureBankingAccessToolkit/issues/new/choose
+    url: https://github.com/https://github.com/SecureApiGateway/SecureApiGateway/issues/new/choose
```
Comment (shaunharrisonFR) [inline .github/ISSUE_TEMPLATE/config.yml:7]: Double github url

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 blank_issues_enabled: false
 contact_links:
   - name: Developers and Collaborators guidelines
-    url: https://github.com/SecureBankingAccessToolkit/SecureBankingAccessToolkit/wiki/Developers-guideline
+    url: https://github.com/https://github.com/SecureApiGateway/SecureApiGateway/wiki/Developers-guideline
```
[the remark this answers is not in the document]
Comment (jorgesanchezperez) [reply .github/ISSUE_TEMPLATE/config.yml]: Fixed!

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (jorgesanchezperez) [reply .github/ISSUE_TEMPLATE/config.yml]: Fixed

File: secure-api-gateway-ob-uk-ui-common/projects/authentication/package.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+<<<<<<<< Updated upstream:secure-api-gateway-ob-uk-ui-common/projects/authentication/package.json
+  "name": "@secureapigateway/secure-api-gateway-ob-uk-ui-common/authentication",
+========
+  "name": "@secureapigateway/secure-api-gateway-ob-uk-ui-common/oidc",
+>>>>>>>> Stashed changes:secure-api-gateway-ob-uk-ui-common/projects/oidc/package.json
```
Comment (dbadham-fr) [inline secure-api-gateway-ob-uk-ui-common/projects/authentication/package.json:6]: Resolve git merge

Comment (jorgesanchezperez) [reply secure-api-gateway-ob-uk-ui-common/projects/authentication/package.json]: Done

**Review by dbadham-fr — APPROVED** [review]

**Pull request merged**